luna/gateware/soc/simplesoc.py

```diff
@@ -28,13 +28,13 @@ class SimpleSoC(CPUSoC, Elaboratable):
     """ Class used for building simple, example system-on-a-chip architectures.
 
     Intended to facilitate demonstrations (and very simple USB devices) by providing
-    a wrapper that can be updated as the nMigen-based-SoC landscape changes. Hopefully,
-    this will eventually be filled by e.g. nMigen-compatible-LiteX. :)
+    a wrapper that can be updated as the Amaranth HDL-based-SoC landscape changes. Hopefully,
+    this will eventually be filled by e.g. Amaranth HDL-compatible-LiteX. :)
```

Comment:
I think these two instances should be `Amaranth-based-SoC` and `Amaranth-compatible-LiteX`.